Assets/Scripts/Globals.cs

```diff
@@ -81,7 +81,8 @@ public enum EnemyType
 public enum EnemyVariantType
 {
     NONE,
-    HEALER
+    HEALER,
+	SET
```

Comment:
This doesn't look different on my screen... then I realized, someone's using FOUR SPACES instead of a tab. MAX?!

Reply:
damn who would do such a thing